build.bat

```diff
@@ -334,6 +343,7 @@ if !BUILD_CPP! == 1 (
 	)
 
 	set "ADDITIONAL_DEFINES=RTI_LANGUAGE_CPP_TRADITIONAL"
+	set additional_rti_libs=
```

Comment:
I think there is no need to add this line

Reply:
True, I was doing it because in line 357 I was doing: `set additional_rti_libs=nddssecurity !additional_rti_libs!` so I wanted to initialize `additional_rti_libs` somehow. I can safely remove it.